assets/differentialabundance_report.Rmd

```diff
@@ -896,24 +896,28 @@ for (i in 1:nrow(contrasts)){
             contrast_de <- round_dataframe_columns(contrast_de, digits=params$report_round_digits)
             print( htmltools::tagList(datatable(contrast_de, caption = paste('Differential genes', dir, 'in', contrast_descriptions[i], " (check", differential_files[[i]], "for more detail)"), rownames = FALSE) ))
 
-        # Plot Differentially Expressed Genes by Gene Biotype
-        gene_biotype_table <- contrast_de %>%
-            group_by(`Gene biotype`) %>%
-            summarise(count = dplyr::n(), .groups = 'drop') %>%
-            filter(count > 0) %>%
-            arrange(desc(count))
-
-        gene_biotype_plot <- ggplot(gene_biotype_table, aes(x = reorder(`Gene biotype`, -count), y = count)) +
-            geom_bar(stat = "identity", position = position_dodge()) +
-            labs(
-                title = paste0("Differentially Expressed Genes by Gene Biotype (", dir, ")"),
-                x = "Gene Biotype",
-                y = "Number of Differentially Expressed Genes"  ) +
-            theme_minimal() +
-            theme(axis.text.x = element_text(angle = 45, hjust = 1))
-
-        print(gene_biotype_plot)
-
+        # Check if Gene biotype column exists
```

Comment:
No all your doing, but could you see if you can make the indenting less mad before we merge this?

Reply:
Thank you for your feedback, now it should be easier to see the changes :)